### PR TITLE
Replaced .count with .size in Line 49

### LINKED ISSUE #8489: Replace .count on app/views/search to .size
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the files where we have `.count` with `.size` . Using .size increases the speed of the query.

1. https://github.com/publiclab/plots2/blob/main/app/views/search/all_content.html.erb#L49
2. https://github.com/publiclab/plots2/blob/main/app/views/search/_questions_blank.html.erb#L20
3. https://github.com/publiclab/plots2/blob/main/app/views/search/_tags_blank.html.erb#L12
4. https://github.com/publiclab/plots2/blob/main/app/views/search/_wikis_blank.html.erb#L21
5. https://github.com/publiclab/plots2/blob/main/app/views/search/_profiles_blank.html.erb#L15

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
$DIFF
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by Annysah**
Fixes #8489 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: app/views/search/all_content.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="col-md-2">
+m <div class="col-md-2">
```
Comment (cesswairimu) [inline app/views/search/all_content.html.erb:1]: Seems like you added this `m` by mistake. Kindly edit the file and remove the `m` at the beginning of the file and commit.
you can find it here https://github.com/Annysah/plots2/blob/patch-6/app/views/search/all_content.html.erb.

**Review by cesswairimu — CHANGES_REQUESTED** [review]
Great job here :100:  almost there

**Comment by RuthNjeri** [comment]
Good job @Annysah, I'm just curious if changing .count to .size also applied to the queries too? Since multiple models use .count.. @cesswairimu can confirm this?

**Comment by RuthNjeri** [comment]
Also, there are some files that have been left out in the view like
[Form Partial] (https://github.com/publiclab/plots2/blob/main/app/views/locations/_form.html.erb#L48)
[Card Partial] (https://github.com/publiclab/plots2/blob/main/app/views/notes/_card.html.erb#L28) and line 29

and some more.. 

If you use Visual Studio Code as an editor, you can search the word `count` and you'll see all the files that reference it, that's what I have done @Annysah, if you are not using Visual Studio Code, I think you can just do a global search within those files, or use the global search of your editor.

Screenshot 2020-10-09 at 21 08 21

**Comment by cesswairimu** [comment]
Yeah it applies to some of them, we have other issues opened for some of these files. sometimes `.size` does  not  replace `.count`  seamlessly  e.g on this line https://github.com/publiclab/plots2/blob/main/app/models/node.rb#L197 using `.size` returns nil  but .count returns 0. That's why replacing a couple of files per issue is crucial so that we don't break existing functionality.  Thanks

**Comment by RuthNjeri** [comment]
It's clear now, thanks @cesswairimu

**Comment by Annysah** [comment]
Hello @cesswairimu thanks for the review, I have removed the m which was mistakenly added.
Can I get a review?

**Comment by Annysah** [comment]
Thanks @RuthNjeri

**Review by Tlazypanda — APPROVED** [review]
@Annysah Looking good :tada: Thanks for working on this!!

**Comment by Annysah** [comment]
Thank you! @Tlazypanda

**Review by cesswairimu — APPROVED** [review]
Awesome all good now :tada:

**Comment by cesswairimu** [comment]
Thanks @Annysah for working on this :rocket:  would you like to close the other pull requests since we updated all files here? 
 Thanks @Tlazypanda and @RuthNjeri  for the reviews :tada:

**Comment by Annysah** [comment]
Oh my!!! I'm so excited that my PR was merged!!!

**Pull request merged**